pkg/etcdcli/health.go

```diff
@@ -44,6 +44,10 @@ func getMemberHealth(ctx context.Context, etcdMembers []*etcdserverpb.Member) me
 	memberHealth := memberHealth{}
 	hch := make(chan healthCheck, len(etcdMembers))
 	for _, member := range etcdMembers {
+		if member.IsLearner {
+			// Learner is not health checked until promoted.
+			continue
+		}
```

Comment:
this requires a func name change. A learner is a member. If you ignore learners call it `getPromotedMemberHealth`.